src/services/component-service.ts

```diff
@@ -102,6 +108,29 @@ export class ComponentService {
     return component
   }
 
+  validateComponent(component: Component<ComponentType>): void {
```

Comment:
@alepintus @zonia3000 General bundle descriptor validation logic is being implemented in https://entando.myjetbrains.com/youtrack/issue/ENG-3788 so please add a TODO to refactor once that is done

Reply:
done